src/org/ods/orchestration/phases/DeployOdsComponent.groovy

```diff
@@ -58,15 +58,24 @@ class DeployOdsComponent {
                     applyTemplates(openShiftDir, deploymentMean)
 
                     def retries = project.environmentConfig?.openshiftRolloutTimeoutRetries ?: 10
+                    def podData = null
                     for (def i = 0; i < retries; i++) {
-                        def podData = os.checkForPodData(project.targetProject, deploymentMean.selector)
+                        podData = os.checkForPodData(project.targetProject, deploymentMean.selector)
                         if (podData) {
                             return podData
                         }
                         steps.echo("Could not find 'running' pod(s) with label '${deploymentMean.selector}' - waiting")
                         steps.sleep(12)
                     }
 
+                    if (podData == null) {
+                        throw new RuntimeException(
```

Comment:
Please, create a specific exception for this, so that it can be caught, if necessary.